flutter/lib/src/event_processor/platform_exception_event_processor.dart

```diff
@@ -0,0 +1,115 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+import 'package:sentry/sentry.dart';
+import 'package:stack_trace_parser/stack_trace_parser.dart';
+
+class AndroidPlatformExceptionEventProcessor implements EventProcessor {
+  const AndroidPlatformExceptionEventProcessor();
+
+  @override
+  FutureOr<SentryEvent?> apply(SentryEvent event, {hint}) async {
+    final plaformException = event.throwable;
+    if (!(plaformException is PlatformException)) {
```

Comment:
```suggestion
    if (plaformException !is PlatformException) {
```

Reply:
This auto-formats to `plaformException! is PlatformException` which looks worse to me.

Reply:
That's strange, the formatter is wrong then?

Reply:
Not sure, I'll give it another look later.

Reply:
Yeah, it looks just worse without paranthesis.

Reply:
let's keep it as it is then :)